var/spack/repos/builtin/packages/ants/package.py

```diff
@@ -15,13 +15,26 @@ class Ants(CMakePackage):
        processing library to which ANTs developers contribute.
     """
 
-    homepage = "http://stnava.github.io/ANTs/"
+    homepage = "https://stnava.github.io/ANTs/"
     url      = "https://github.com/ANTsX/ANTs/archive/v2.2.0.tar.gz"
 
     version('2.2.0', sha256='62f8f9ae141cb45025f4bb59277c053acf658d4a3ba868c9e0f609af72e66b4a')
+    version('2.3.5', sha256='2fddfd5f274a47f1c383e734a7e763b627c4a8383d2d3b9971561f335016bb0a')
 
     depends_on('zlib', type='link')
 
+    variant('MINC', values=bool, default=True,
+            description='Build ITK with MINC support')
+
+    def cmake_args(self):
+        args = []
+        minc_support = bool(self.spec.variants['MINC'].value)
+        if minc_support:
+            args.extend([
+                '-DITK_BUILD_MINC_SUPPORT=ON',
+            ])
+        return args
```

Comment:
We have a convenience method in `CMakePackage` to deal with define set based on a variant value:
```suggestion
        return [
            self.define_from_variant('ITK_BUILD_MINC_SUPPORT', 'minc')
        ]
```

Reply:
All fixed, thanks.